shared/src/commonMain/composeResources/values-de/strings.xml

```diff
@@ -466,4 +467,20 @@
     <string name="text_label_mandatory_attributes">Notwendige Attribute</string>
     <string name="text_label_requests">Anfragen</string>
     <string name="info_text_no_requests">Keine Anfragen</string>
+
+    <!-- Device capabilities -->
+    <string name="capabilities_heading_signing">Signierung</string>
+    <string name="capabilities_heading_attestation">Attestierung</string>
+    <string name="capabilities_heading_internet">Internet</string>
+
+    <string name="info_text_capabilities_no_internet">Bitte prüfen Sie Ihre Internetverbindung und aktivieren Sie in den Systemeinstellungen gegebenenfalls mobile Daten oder deaktivieren Sie den Flugmodus.</string>
+    <string name="info_text_capabilities_no_signing">Auf diesem Gerät ist das Auslösen von Signaturen derzeit nicht möglich. Prüfen Sie in den Einstellungen, ob eine Displaysperre aktiviert und ein Fingerabdruck oder Face ID hinterlegt ist.</string>
+    <string name="info_text_capabilities_no_attestation">Die Attestierung dieses Geräts ist nicht möglich. Möglicherweise schafft ein Betriebssystem-Update Abhilfe.</string>
+    <string name="info_text_capabilities_no_attestation_ios">In manchen Fällen ist der Attestation Service von Apple nicht erreichbar.</string>
```

Comment:
Maybe rephrase to add "check your internet connection / VPN" or something like that?!